src/sinol_make/commands/run/__init__.py

```diff
@@ -258,9 +260,11 @@ def configure_subparser(self, subparser):
                             help=f'tool to measure time and memory usage (default: {default_timetool})')
         parser.add_argument('--oiejq-path', dest='oiejq_path', type=str,
                             help='path to oiejq executable (default: `~/.local/bin/oiejq`)')
-        add_compilation_arguments(parser)
+        parser.add_argument('-p', '--print-expected-scores', dest='print_expected_scores', action='store_true',
+                            help='print whole expected scores report')
```

Comment:
Maybe we don't need this flag at all? After all, we see how many points each solution got in the table above it.

Reply:
Do you mean that we should completely abandon printing `sinol_expected_scores`? I think that they aren't needed anymore, since the table prints the same things and the messages about changes should catch all changes.

Reply:
Hmm, but it may be useful if someone wanted to compare results with config easly